tests/symbolicator/snapshots/SymbolicatorMinidumpIntegrationTest/test_full_minidump.pysnap

```diff
@@ -190,17 +216,38 @@ exception:
       - in_app: false
         instruction_addr: '0x750662c4'
         package: C:\Windows\System32\kernel32.dll
-        trust: cfi
+        trust: fp
       - abs_path: f:\dd\vctools\crt\vcstartup\src\startup\exe_common.inl
+        data:
+          symbolicator_status: symbolicated
         filename: exe_common.inl
         function: __scrt_common_main_seh
         in_app: false
-        instruction_addr: '0x2a2d97'
+        instruction_addr: '0x2a2d96'
         lineno: 283
         package: C:\projects\breakpad-tools\windows\Release\crash.exe
         symbol: __scrt_common_main_seh
-        trust: cfi
+        trust: scan
+      - abs_path: f:\dd\vctools\crt\vcstartup\src\utility\utility_desktop.cpp
+        data:
+          symbolicator_status: symbolicated
+        filename: utility_desktop.cpp
+        function: __scrt_set_unhandled_exception_filter
+        in_app: false
+        instruction_addr: '0x2a3434'
+        lineno: 84
+        package: C:\projects\breakpad-tools\windows\Release\crash.exe
+        symbol: __scrt_set_unhandled_exception_filter
+        trust: scan
+      - data:
+          symbolicator_status: symbolicated
```

Comment:
Why is this symbolicated but does not have any function name?

Reply:
Ah, this happens when symbolic does not return line info and we reinsert the raw frame. Maybe we want our own status for this at some point. cc @jan-auer

Reply:
Replaced with missing_symbol, because of https://sentry.io/organizations/sentry/issues/1014119863

commit: https://github.com/getsentry/symbolicator/commit/08c0fcc9da4f8b9c52ec6238e2bcae841e3949ef